t/unittest/condition/key.t

```diff
@@ -22,18 +22,18 @@ use Test::More tests => 7;                      # last test to print
 
 use Test::NoWarnings;
 
-use_ok("pf::profile::filter::key");
+use_ok("pf::condition::key");
```

Comment:
condition key doesn't exist anymore. will need to check if there is a test for equals. if yes we'll remove this test